CommandHelper/LocalPackages/commands/chars/madhatterjervistetch.yml

```diff
@@ -1,9 +1,9 @@
 class: madhatterjervistetch
 costumes:
 - chatcolor: '&6'
-  name: &2M&aad&dHa&5tt&ee&6r 
```

Comment:
His name needs (JT). If the name is specified in the class/filename, it should be specified in his displayname as initials